decisions/wc_offmap_decisions.txt

```diff
@@ -381,14 +427,18 @@ offmap_decisions = {
 			}
 		}
 	}
-	request_army_offmap_yoggsaron = {
+	request_general_offmap = {
```

Comment:
I would comment it out since it's not used.

Reply:
it will be changed in another branch